.nvmrc

```diff
@@ -1 +1 @@
-v14.17.0
+v18.0.0
```

Comment:
Is this change needed?

Reply:
> Is this change needed?

If this file will not have the current version (as in package.json) then why do I need this file?

Reply:
> Is this change needed?

If this file will not have the current version (as in [package.json](https://github.com/jbx-protocol/juice-interface/blob/main/package.json#L59)) then why do I need this file?